xtable-hive-metastore/src/main/java/org/apache/xtable/hms/HMSCatalogConfig.java

```diff
@@ -43,6 +45,15 @@ public class HMSCatalogConfig {
   @JsonProperty("externalCatalog.hms.serverUrl")
   private final String serverUrl;
 
+  @JsonProperty("externalCatalog.hms.schema_string_length_thresh")
+  private final int schemaLengthThreshold = 4000;
+
+  @JsonProperty("externalCatalog.hms.partition_extractor_class")
+  private final String partitionExtractorClass = MultiPartKeysValueExtractor.class.getName();
+
+  @JsonProperty("externalCatalog.hms.max_partitions_per_request")
+  private final int maxPartitionsPerRequest = 1000;
+
```

Comment:
This can be moved to a `HMSPartitionSyncConfig` ? It would be populated by formats which sync partitions to the catalog.